tox.ini

```diff
@@ -16,6 +16,10 @@ setenv =
 requires =
     pip>=19.2.1
 
+[testenv:py37]
+requires =
+    venusian
```

Comment:
I would recommend adding this to `setup.py` with an environment marker for `python_version>=3.7`

Reply:
Presumably you mean on this line:

```python
tests_require = ['pytest', 'pytest-cov', 'venusian', 'sybil']
```

How do I add venusian as a dependency with the python_version environment marker?

Reply:
`venusian; python_version >= 3.7`

Reply:
https://www.python.org/dev/peps/pep-0496/